### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by robinpiets**
Added unit tests for:
- atoms/Alert
- atoms/ComponentWrapper
- atoms/EditButton
- atoms/List
- atoms/PrintOnly
- atoms/TextToEdit
- components/AddressLine
- components/ContactSentence
- components/PrintDetails
- components/QuestionAnswer
- components/RegisterLookupSummary
- DefaultLayout

**Please Squash and merge (-:**

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [ ] ~you followed our checklist for functional testing~
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: apps/client/src/atoms/atoms.test.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import "jest-styled-components";
```
Comment (afjlambert) [inline apps/client/src/atoms/atoms.test.js]: Can you split this up into 1 test-file per component please?

File: apps/client/src/components/components.test.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
+import "jest-styled-components";
```
Comment (afjlambert) [inline apps/client/src/components/components.test.js]: Here too, please split into separate test-files.

File: apps/client/src/components/components.test.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+let onClickMock;
+
+afterEach(() => {
+  onClickMock = jest.fn();
+
+  cleanup();
+});
+
+// These tests are meant to test only the most basic components. Advanced components need their own file.
```
Comment (afjlambert) [inline apps/client/src/components/components.test.js]: I understand but this is not very portable. Also you need to do more work when a component grows/shrinks, is moved to `atoms` for example.

File: apps/client/src/utils/index.js [hunk 4]
```diff
@@ -2,6 +2,11 @@ import isBoolean from "lodash.isboolean";
 import isNumber from "lodash.isnumber";
 import isString from "lodash.isstring";
 
+import { topics } from "../config";
+
+// Find a topic by the slug
+export const findTopicBySlug = (slug) => topics.find((t) => t.slug === slug);
```
Comment (afjlambert) [inline apps/client/src/utils/index.js:8]: Small test for this one would be very nice.

**Review by afjlambert — CHANGES_REQUESTED** [review]
Overall looks great! Please split the test-files.

**Review by svenjens — APPROVED** [review]

File: apps/client/src/utils/index.js (hunk 4 above)
Comment (robinpiets) [reply apps/client/src/utils/index.js:8]: It has a test, but it is on the [Conclusion PR](https://github.com/Amsterdam/vergunningcheck/pull/522/files). This is a PR to develop

File: apps/client/src/components/components.test.js (hunk 3 above)
Comment (robinpiets) [reply apps/client/src/components/components.test.js]: Done, thanks for the feedback

**Pull request merged**